.github/workflows/main.yml

```diff
@@ -0,0 +1,147 @@
+        tar -xvf kubebuilder_${{env.KUBEBUILDER_VERSION}}_linux_amd64.tar.gz
+        mv kubebuilder_${{env.KUBEBUILDER_VERSION}}_linux_amd64 /usr/local/kubebuilder
+
+    - name: Test
+      run: make test
+
+    - name: Coverage
+      uses: codecov/codecov-action@v1
+      with:
+        # token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
+        file: ./cover.out
+        # flags: unittests # optional
+        name: externalsecret-operator
+        fail_ci_if_error: true 
+
+  docker:
+    name: Docker
+    runs-on: ubuntu-latest
+    needs: [build, test]
+    steps:
+    - name: Prepare
+      id: prep
+      run: |
+        DOCKER_IMAGE=ghcr.io/external-secrets/external-secrets
+
+        VERSION=edge
+        if [[ $GITHUB_REF == refs/tags/* ]]; then
+          VERSION=${GITHUB_REF#refs/tags/}
+        elif [[ $GITHUB_REF == refs/heads/* ]]; then
+          VERSION=$(echo ${GITHUB_REF#refs/heads/} | sed -r 's#/+#-#g')
+        elif [[ $GITHUB_REF == refs/pull/* ]]; then
+          VERSION=pr-${{ github.event.number }}
+        fi
+
+        TAGS="${DOCKER_IMAGE}:${VERSION}"
+        if [ "${{ github.event_name }}" = "push" ]; then
+          TAGS="$TAGS,${DOCKER_IMAGE}:sha-${GITHUB_SHA::8}"
+        fi
+
+        PUSH_IMAGE=true
+        REPO_FULL_NAME="${{ github.event.pull_request.head.repo.full_name }}"
+        # If this is both a pull request and a fork, then don't push the image
+        if [[ ${{ github.event_name }} == pull_request ]]; then
+          if [[ $REPO_FULL_NAME != external-secrets/external-secrets ]]; then
+            PUSH_IMAGE=false
+          fi
+        fi
+
+        echo ::set-output name=version::${VERSION}
+        echo ::set-output name=tags::${TAGS}
+        echo ::set-output name=created::$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+        echo ::set-output name=push_image::$PUSH_IMAGE
+
+    - name: Check out the repo
+      uses: actions/checkout@v2
+      
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: all
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      
+    - name: Login to Github Packages
+      id: docker-login
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ secrets.GHCR_USERNAME }}
+        password: ${{ secrets.GHCR_TOKEN }}
+      if: ${{ steps.prep.outputs.push_image == 'true' }}
+
+    - name: Build and push
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        builder: ${{ steps.buildx.outputs.name }}
+        platforms: linux/amd64
+        tags: ${{ steps.prep.outputs.tags }}
+        push: ${{ steps.prep.outputs.push_image }}
+        labels: |
+          org.opencontainers.image.source=${{ github.event.repository.clone_url }}
```

Comment:
Github seems to want some other URL 
![image](https://user-images.githubusercontent.com/326929/101814682-c380fc00-3b1e-11eb-8ae1-afc775481af1.png)
It works regardless but would be nice if the integration connected :)